exampleSite/content/encounters/uwu.md

```diff
@@ -0,0 +1,15 @@
+---
+title: The Weapon's Refrain
```

Comment:
We'll probably need to make a few changes in the editor here, but it's not so critical that I would block right now:

1. The title should be the actual full title of the fight (so that `.Title` things can pick it up for SEO)
2. We'll probably need to have a "short name" which will be the file name/route fragement name (see [here](https://github.com/The-Balance-FFXIV/balance-static/blob/8c603d2545795e5ae5a71fa613d1a13cae481657/static/admin/config.yml#L48) for how to override the title to a name)